src/libcore/fmt/mod.rs

```diff
@@ -298,7 +298,7 @@ impl<'a> Display for Arguments<'a> {
 ///
 /// For more information on formatters, see [the module-level documentation][module].
```

Comment:
Ah, I was looking at http://doc.rust-lang.org/nightly/std/fmt/trait.Display.html, which with my patch winds up linking to http://doc.rust-lang.org/nightly/std/fmt/index.html. I didn't realize the docs would wind up in two places.